llvm/lib/Target/SBF/SBFFrameLowering.cpp

```diff
@@ -16,24 +16,23 @@
 #include "llvm/CodeGen/MachineFrameInfo.h"
 #include "llvm/CodeGen/MachineFunction.h"
 #include "llvm/CodeGen/MachineInstrBuilder.h"
-#include "llvm/CodeGen/MachineRegisterInfo.h"
 
 using namespace llvm;
 
 namespace {
 
 void adjustStackPointer(MachineFunction &MF, MachineBasicBlock &MBB,
                         MachineBasicBlock::iterator &MBBI,
-                        unsigned int Opcode) {
+                        bool IsSubtract) {
   MachineFrameInfo &MFI = MF.getFrameInfo();
   int NumBytes = (int)MFI.getStackSize();
   if (NumBytes) {
     DebugLoc Dl;
     const SBFInstrInfo &TII =
         *static_cast<const SBFInstrInfo *>(MF.getSubtarget().getInstrInfo());
-    BuildMI(MBB, MBBI, Dl, TII.get(Opcode), SBF::R11)
+    BuildMI(MBB, MBBI, Dl, TII.get(SBF::ADD_ri), SBF::R11)
         .addReg(SBF::R11)
-        .addImm(NumBytes);
+        .addImm(IsSubtract? -NumBytes : NumBytes);
```

Comment:
Reminder: please be mindful of the LLVM coding conventions[*]. You can run `clang-format` to help identify/automate correct coding conventions. Upstream LLVM runs this automatically for submitted patches, but we do not yet do this in our local CI.
Also, when you run `clang-format` on future patches, only commit the diffs that are within the lines of your patch.

[*] See the links in, e.g., https://llvm.org/docs/Contributing.html#how-to-submit-a-patch

Reply:
Should I keep a single commit in the PR?

Reply:
Yes, you should do a squash commit (with updated log message for the full commit) when done. The github UI will ask you.